utils/dbfixup.py

```diff
@@ -51,57 +51,153 @@ def bits_str(bits):
     return ' '.join(sorted(list(bits)))
 
 
-def zero_groups(tag, bits, zero_db, strict=True, verbose=False):
-    """
-    See if a line occurs within a bit group
-    If it does, add 0 bits
+class ZeroGroups(object):
+    def __init__(self, zero_db):
+        self.groups = []
+        self.bit_to_group = {}
+        self.tag_to_groups = {}
+        self.zero_tag_to_group = {}
+        self.parse_zero_db(zero_db)
 
-    Ex: 01_02 04_05
-    Means find a line that has either of these bits
-    If either of them occurs, default bits in that set to zero
+    def print_groups(self):
+        print('Zero groups:')
+        for bits in self.groups:
+            print(bits_str(bits))
 
-    Ex: 01_02 04_05|07_08 10_11
-    If any bits from the first group occur,
-    default bits in the second group to zero
+        print('Zero tags:')
+        for tag in self.zero_tag_to_group:
+            print(tag, bits_str(self.zero_tag_to_group[tag]))
 
-    Ex: 01_02 04_05,ALL_ZERO
-    ALL_ZERO is an enum that is part of the group but is all 0
-    It must have 0 candidates
+    def parse_zero_db(self, zero_db):
+        """ Convert zero db format into data structure
 
-    strict: assert that the size of the given group is the size of the given mask
-    """
-    for zdb in zero_db:
-        allzero_tag = None
-        if "," in zdb:
-            zdb, allzero_tag = zdb.split(",")
-
-        if "|" in zdb:
-            a, b = zdb.split("|")
-            a = a.split()
-            b = b.split()
-        else:
-            a = zdb.split()
-            b = a
+        Zero db format examples:
```

Comment:
So, I know you didn't come up with this, but I do find this syntax quite confusing....

Reply:
Would you like to be more specific about what is confusing, it doesn't seem to bad.  This comment is not actionable, and I'm not planning on changing every dbf file over a vague comment.